homeassistant/helpers/logging.py

```diff
@@ -0,0 +1,49 @@
+"""Helpers for logging allowing more advanced logging styles to be used."""
+import inspect
+import logging
+
+
+class BraceMessage:
+    """
+    Represents a logging message with brace style arguments.
```

Comment:
What are brace style arguments? Keyword arguments?

Reply:
Is that an established lingo for this case? I was confused. `BraceMessage` to me sounds like a message that we provide to brace something.

Reply:
Yes, I like that better. :+1: